Missing some enum value identifiers in the OdooModelMapper.GetDotNetModel

When generating the .NET models for some models, some enums are generated with a missing identifier for the first enum value.
I would expect the identifier to be called `None` for example to express the fact that none of the following options are selected.

### Example of PayrollResUsersOdooEnum
````
/// <summary>
/// </summary>
[JsonConverter(typeof(StringEnumConverter))]
public enum PayrollResUsersOdooEnum
{
[EnumMember(Value = "False")]
 = 1,                                                                    // error CS1001: Identifier expected

[EnumMember(Value = "80")]
Officer = 2,

[EnumMember(Value = "81")]
Manager = 3,
}

````

### List of impacted enums

I got this for the following enums :

- SalesResUsersOdooEnum
- PayrollResUsersOdooEnum
- WebsiteResUsersOdooEnum
- EmployeesResUsersOdooEnum
- InventoryResUsersOdooEnum
- AdministrationResUsersOdooEnum
- LeavesResUsersOdooEnum
- PurchasesResUsersOdooEnum
- EmailMarketingResUsersOdooEnum
- AccountingFinanceResUsersOdooEnum
- RecruitmentResUsersOdooEnum
- ManufacturingResUsersOdooEnum
- FleetResUsersOdooEnum
- PointOfSaleResUsersOdooEnum


----

Odoo version : v16
.NET version : .NET Core 6
Platform : Mac M1

--- a/PortaCapena.OdooJsonRpcClient/Converters/OdooModelMapper.cs
+++ b/PortaCapena.OdooJsonRpcClient/Converters/OdooModelMapper.cs
@@ -219,7 +219,13 @@ namespace PortaCapena.OdooJsonRpcClient.Converters
         {
             odooName = odooName.Replace("+", "Plus");
             var odooNameCleaned = Regex.Replace(odooName, "[^A-Za-z0-9-]", "_");
-            var dotnetKeys = odooNameCleaned.Split('_', '-', '.', ',', ' ', ':', ';', '/', '\\', '*', '+', '(', ')', '[', ']').Select(x => CultureInfo.InvariantCulture.TextInfo.ToTitleCase(x));
+            var dotnetKeys = odooNameCleaned
+                .Split('_', '-', '.', ',', ' ', ':', ';', '/', '\\', '*', '+', '(', ')', '[', ']')
+                .Select(x => CultureInfo.InvariantCulture.TextInfo.ToTitleCase(x))
+                .Where(x => !string.IsNullOrEmpty(x))
+                .ToList();
+            if (dotnetKeys.Count == 0)
+                dotnetKeys.Add("None");
             return string.Join(string.Empty, dotnetKeys);
         }
 

